### DISCUSSION TITLE
One rule with different/multiple severities

### DISCUSSION THREAD

**Discussion by krnlde** [eslint/eslint#14565] [Q&A]
## Use case
I want to be able to have different severities for different warning comments.

### .eslintrc.js
```js
module.exports = {
    // ...
    'no-warning-comments': ['error', { terms: ['fixme'], location: 'start' }],
    'no-warning-comments': ['warn', { terms: ['todo', 'wip'], location: 'start' }],
    // ...
};
```

Of course in the above example only the last rule gets recognized. Therefor I already tried different approaches which won't work either:
### approach 2
```js
module.exports = {
    // ...
    'no-warning-comments': [
      'error', { terms: ['fixme'], location: 'start' },
      'warn', { terms: ['todo', 'wip'], location: 'start' }
    ],
    // ...
};
```

### approach 3
```js
module.exports = {
    // ...
    'no-warning-comments': [
      ['error', { terms: ['fixme'], location: 'start' }],
      ['warn', { terms: ['todo', 'wip'], location: 'start' }]
    ],
    // ...
};
```
I already searched the entire interwebs for "eslint duplicate rules", "eslint rule different severities", and alike.
Got not match in stackoverflow, in the issues here and in the discussions.

Any suggestions on how to solve that? Is this really this edge-casey?

Cheers,
Kai

**Comment by aladdin-add** [comment]
atm, eslint does not support this kind of thing. :)
